xml/System.Numerics/Complex.xml

```diff
@@ -6983,8 +6983,7 @@ Languages that support custom operators can use the <xref:System.Numerics.Comple
       <MemberSignature Language="F#" Value="abstract member TryFormat : Span&lt;byte&gt; * int * ReadOnlySpan&lt;char&gt; * IFormatProvider -&gt; bool&#xA;override this.TryFormat : Span&lt;byte&gt; * int * ReadOnlySpan&lt;char&gt; * IFormatProvider -&gt; bool" Usage="complex.TryFormat (utf8Destination, bytesWritten, format, provider)" />
       <MemberType>Method</MemberType>
       <Implements>
-        <InterfaceMember>M:System.Numerics.INumberBase`1.TryFormat(System.Span{System.Byte},System.Int32@,System.ReadOnlySpan{System.Char},System.IFormatProvider)</InterfaceMember>
-        <InterfaceMember>M:System.IUtf8SpanFormattable.TryFormat(System.Span{System.Byte},System.Int32@,System.ReadOnlySpan{System.Char},System.IFormatProvider)</InterfaceMember>
+        <InterfaceMember>M:System.Numerics.INumberBase`1.System#IUtf8SpanFormattable#TryFormat(System.Span{System.Byte},System.Int32@,System.ReadOnlySpan{System.Char},System.IFormatProvider)</InterfaceMember>
```

Comment:
@huangmin-ms This line is causing the following warning:

Line 0: [Warning] Cross reference not found: 'System.Numerics.INumberBase`1.System'.

This seems like a bug. I'm not sure where INumberBase`1 comes into play here?

Reply:
@gewarren  Actually this diff is expected.  It was caused by `TryFormat` method of `INumberBase` changed from a normal method to an EII member. 
https://github.com/dotnet/runtime/blob/18cac43e79503cb2d9424dd19822e7ee8444db26/src/libraries/System.Private.CoreLib/src/System/Numerics/INumberBase.cs#L472

The original inherit chain is: `Complex.TryFormat()` -> `INumberBase.TryFormat()` -> `IUtf8SpanFormattable.TryFormat()`
And now becomes: `Complex.TryFormat()` -> `INumberBase.IUtf8SpanFormattable.TryFormat()`

The reason why you see the xref warning is because there's `#` in EII signature and currently our build system is not able to handle that character properly.
I logged to bug https://ceapex.visualstudio.com/Engineering/_workitems/edit/926457 to track this issue.

Is this warning a blocking issue for this PR to get merged? or can we ignore it for now?

Reply:
@tannergooding Is this what you'd expect to see here, i.e. that `Complex.TryFormat()` implements `INumberBase<T>.IUtf8SpanFormattable.TryFormat()` and not `IUtf8SpanFormattable.TryFormat()`?

Reply:
@gewarren  Convert  `#` to `%23` in xml file will be overwritten in next pipeline run.  We will escape it in `Ecma2Yaml` tool.